frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -51,6 +52,7 @@ const DeployRegisteredModelModal: React.FC<DeployRegisteredModelModalProps> = ({
   const [dataConnections] = useDataConnections(selectedProject?.metadata.name);
   const [connections] = useConnections(selectedProject?.metadata.name, true);
   const error = platformError || projectError;
+  const [rm] = useRegisteredModelById(modelVersion.registeredModelId);
```

Comment:
Same deal here with the loading/error states. We should combine the loading/error here with `deployInfoLoaded` and `deployInfoError` below.